content/release/latest-stable-release-sabayon-1903.md

```diff
@@ -0,0 +1,125 @@
+title = "Sabayon 19.03 - New stable release"
+description = "19.03 Press release"
+date = "2019-03-31"
+categories = ["releases", "sabayon"]
+tags = ["releases", "sabayon"]
++++
+
+The team behind Sabayon is excited to present you the latest stable release:
+Sabayon 19.03.
+
+Sabayon is a modern and easy to use distribution based on Gentoo,
+which follows a reliable rolling release model.
+
+Please read on or [download](/download/) your flavour :-)
+
+19.03 is a long awaited release, coming with a lot of new features and enhancements:
+
+- New build infrastructure
+- Switch to Dracut for initramfs generation
+- Full disk encryption support
+- We changed our default installer. We dropped Anaconda, Long live to Calamares!
+- Added UEFI32 bit support! (Intel Sticks, and so on..)
+- Latest kernel 4.20
+- Python 3 as default
```

Comment:
Maybe mentioning the minor release as well?

Reply:
I think that this information is related to kernel available on ISO image that is 4.20. Any other information could be retrieved by users with `equo search` command